events/management/commands/check_deprecated_yso_keywords.py

```diff
@@ -0,0 +1,88 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from events.importer import yso
+from events.models import Keyword
+
+
+class DryRun(Exception):
+    pass
+
+
+class Command(BaseCommand):
+    help = "Check replacements of deprecated keywords."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--apply",
+            action="store_true",
+            help="Apply changes",
+        )
+
+    def handle(self, **options):
+        apply = options["apply"]
+        importer = yso.YsoImporter({})
+        importer.setup()
+        self.stdout.write("Loading graph into memory, this can take a while..")
```

Comment:
[An ellipsis consists of three periods.](https://en.wikipedia.org/wiki/Ellipsis)
```suggestion
        self.stdout.write("Loading graph into memory, this can take a while...")
```